lib/pages/emergency_pass.dart

```diff
@@ -0,0 +1,142 @@
+                            ),
+                            onTap: () => {
+                              showDialog(
+                                  context: context,
+                                  builder: (BuildContext context) {
+                                    return AlertDialog(
+                                      title: Text(AppLocalizations.of(context)!.organDonor),
+                                      actions: [
+                                        TextButton(
+                                          child: Text(AppLocalizations.of(context)!.yes),
+                                          onPressed: () {
+                                            setState(() {
+                                              organDonor = AppLocalizations.of(context)!.yes;
+                                            });
+                                            Navigator.of(context).pop();
+                                          },
+                                        ),
+                                        TextButton(
+                                          child: Text(AppLocalizations.of(context)!.no),
+                                          onPressed: () {
+                                            setState(() {
+                                              organDonor = AppLocalizations.of(context)!.no;
+                                            });
+                                            Navigator.of(context).pop();
+                                          },
+                                        ),
+                                      ],
+                                    );
+                                  })
+                            }
+                          ),
+                        ),
+
+                        Padding(padding: const EdgeInsets.symmetric(vertical: paddingSmall),
+                          child: TextField(
+                            decoration: InputDecoration(
+                              border: const OutlineInputBorder(),
+                              prefixIcon: const Icon(Icons.monitor_weight),
+                              labelText: AppLocalizations.of(context)!.weight,
+                              hintText: AppLocalizations.of(context)!.weight,
+                            ),
+                          ),
+                        ),
+                        Padding(padding: const EdgeInsets.symmetric(vertical: paddingSmall),
+                          child: TextField(
+                            decoration: InputDecoration(
+                              border: const OutlineInputBorder(),
+                              prefixIcon: const Icon(Icons.height),
+                              labelText: AppLocalizations.of(context)!.height,
+                              hintText: AppLocalizations.of(context)!.height,
+                            ),
+                          ),
+                        ),
```

Comment:
weight, height should be [rangesliders](https://api.flutter.dev/flutter/material/RangeSlider-class.html) or have only number, float input. The unit of measurement like meter or feet would be nice in a suffix.